common/search/src/main/scala/weco/api/search/elasticsearch/ElasticsearchService.scala

```diff
@@ -72,6 +77,37 @@ class ElasticsearchService(elasticClient: ElasticClient)(
         }
     }
 
+  def executeMultiSearchRequest(
+    request: MultiSearchRequest
+  ): Future[Either[ElasticsearchError, MultiSearchResponse]] =
+    spanFuture(
+      name = "ElasticSearch#executeMultiSearchRequest",
```

Comment:
@jamieparkinson i'm wondering if this is the right way to do this? I'm guessing this will distinguish multiSearchRequests in APM.

The other thing is that i'm accumulating total time to return all the requests in a multi-request, when maybe we want both accumulated and individual time?

Reply:
Think this is right - to be honest it might be worth revisiting the tracing stuff as I did it a long time ago and I think the [state](https://github.com/elastic/apm-agent-java/pull/1048) of scala support has improved a bit since then.

Anyway, re the timing, I would agree that we should capture the individual times (maybe only those, as the total is just derived from those, and the total is maybe a bit misleading as [I think?] the requests can run in parallel).

iirc you can set whatever you want as keys in `transaction.setLabel`; I don't think `elasticTook` isn't special or defined elsewhere.